### DISCUSSION TITLE
Loader / Spinner in cart page

### DISCUSSION THREAD

**Discussion by eskruolis** [addonify/addonify-wishlist#174] [General]
Hi, 

I was wondering if in future updates you could add spinner/loader to wishlist page when button remove fro wishlist or add to cart is clicked. 

I can see that in demo page you provide don't need to do so simply because your admin-ajax call request is 200ms which is unbelievable. , for me this call is 1.5s and it does not look great when item disappears only after 1.5s without loader. Thanks.

Would also want to suggest these functionalities:

1. Ajax update to wishlist item count.
2. Cart items NOT displaying for different languages ( Add Title display with WPML support ).
3. Add remove all button to wishlist page.
4. Add add all product button to wishlist page.
5. Add hook location where to display button in single product page. 
6.  Add possibility to remove item from wishlist if clicked on item which is already in wishlist.
7.  Possibility to add custom icon for wishlist / already in wishlist.

**Comment by Asok17** [comment]
Hello @eskruolis,

Thank you for using `Addonify Wishlist` plugin and providing us the feedback.

In the future update, we'll definitely add the spinner/loader in the Wishlist page, when a product gets removed from the wishlist or is added into the cart.

We greatly appreciate the time and effort that you have spend to share your suggestions regarding the functionalities in the plugin.

We'll create issue(s) for the activities regarding the plugin updates that we'll be pushing in the future. You can find them on [Issues](https://github.com/addonify/addonify-wishlist/issues/) page. We'll notify you here in this discussion once we have completed any issues.

Again, thank you for your feedback and suggestions.

**Comment by h1dd3nsn1p3r** [comment]
Dear @eskruolis 

Happy to have your suggestions. We'll implement them shortly. 👍

**Comment by h1dd3nsn1p3r** [comment]
Implemented in version 1.1.2. Cheers :)
